### DISCUSSION TITLE
point_0710

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#264] [Roy]
"20200307T131241_20200307T131239_T23KNT"![image](https://user-images.githubusercontent.com/54947195/147594922-32191957-924e-4e92-a5b8-e86c35677ed3.png)![image](https://user-images.githubusercontent.com/54947195/147593833-9749c8fa-1e74-4f10-be76-9c582f38855f.png)

"20191128T131241_20191128T131242_T23KNT"
![image](https://user-images.githubusercontent.com/54947195/147594263-0443d97e-b1ce-4754-820b-dd59350299a8.png)
![image](https://user-images.githubusercontent.com/54947195/147594493-536ed4bd-e2ab-4c20-a883-4f886ce44deb.png)

"20190402T131241_20190402T131243_T23KNT"
![image](https://user-images.githubusercontent.com/54947195/147594799-8c8f6550-1386-4e8b-9f0d-ae5a884a7ca5.png)
![image](https://user-images.githubusercontent.com/54947195/147594922-32191957-924e-4e92-a5b8-e86c35677ed3.png)

"20190313T131241_20190313T131238_T23KNT"
![image](https://user-images.githubusercontent.com/54947195/147594977-f1512a1c-f062-41a7-82a6-8c945ce974c3.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200307T131241_20200307T131239_T23KNT"
![Captura](https://user-images.githubusercontent.com/86502862/153330733-0bbac330-0e8a-4427-94ed-987319e375e2.JPG)

**Comment by AntoniCesar** [comment]
"20191128T131241_20191128T131242_T23KNT"
![Captura](https://user-images.githubusercontent.com/86502862/153342730-85111680-3124-4008-aaf2-e6215d52cfdd.JPG)
